CLI/Repl.cpp

```diff
@@ -485,12 +497,30 @@ static void runReplImpl(lua_State* L)
     }
 }
 
+#ifndef _WIN32
+#include <stdexcept>
+void ihandler(lua_State* L, int k) {
+    if (sigint_received) {
+        sigint_received = 0;
+        std::runtime_error error("Execution interrupted");
```

Comment:
This should be `luaL_error(L, "Execution interrupted");` (compatible with building Luau without use of exceptions)

Reply:
So, I tried replacing the exception with a call to luaL_error in my most recent commit. This works fine as a drop-in replacement in the release build, but results in the following error on Ctrl-C in the debug build:
```
VM/src/lapi.cpp(600): ASSERTION FAILED: L->top < L->ci->top
Illegal instruction
```

Is this behavior expected? or should I file an issue?

Reply:
This required `lua_rawcheckstack(L, 1);` as inside the interrupt callback the LUA_MINSTACK guarantee is (intentionally) not upheld.